src/applications/personalization/profile-2/constants.js

```diff
@@ -1,7 +1,17 @@
-export const imagePaths = {
-  Army: '/img/vic-army-symbol.png',
-  'Coast Guard': '/img/vic-cg-emblem.png',
-  'Air Force': '/img/vic-air-force-coat-of-arms.png',
-  Navy: '/img/vic-navy-emblem.png',
-  'Marine Corps': '/img/vic-usmc-emblem.png',
-};
+// The values of these constants map to the possible values that come back from
+// the GET profile/service_history API.
+export const USA_MILITARY_BRANCHES = Object.freeze({
```

Comment:
these constants are used in a couple different places so it made sense to encapsulate them here.